src/Writer/Writer.php

```diff
@@ -60,8 +60,10 @@ class Writer
      * Set plugin loader for use with Extensions
      *
      * @param ExtensionManagerInterface
+     *
+     * @return void
      */
-    public static function setExtensionManager(ExtensionManagerInterface $extensionManager)
+    public static function setExtensionManager(ExtensionManagerInterface $extensionManager): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.